src/test/java/guitests/guihandles/LoanListPanelHandle.java

```diff
@@ -18,6 +18,10 @@ protected String getItemClassName() {
         return Loan.class.getSimpleName();
     }
 
+    protected String getCardPaneId() {
```

Comment:
Id is quite ambiguous as it can mean the ID (eg 4,5 or 6 or LOAN_ID) of the CardPane. Should we use getCardPaneCssId instead?

Reply:
The method name specifically says "CardPaneId". It was the name given in the code previously already, and I don't think there is any ambiguity about what the ID is for (which is the Card itself, not a `Loan` or `Bike` or whatever).

Reply:
I feel that since we are going to include a LoanID, we should try as much as we can to make less ambiguous the names of the methods. I would have to agree with @wn96 on this point.